docs/src/config/services/Systemd.md

```diff
@@ -0,0 +1,35 @@
+# Running spotifyd as a systemd service
```

Comment:
The typo in the filename should also be corrected in the [SUMMARY.md](https://github.com/Spotifyd/spotifyd/blob/603bcc4d802d2b8f45ac517c0513eb4096252d7b/docs/src/SUMMARY.md?plain=1#L15) file,

Reply:
Already done in https://github.com/Spotifyd/spotifyd/commit/b46a3002c3f35b017d33d20104d02a92a2d6f6e5.